### DISCUSSION TITLE
DKIM Prefix Change

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3198] [Development discussions]
Hello,

How can I change the DKIM prefix?

<img width="758" alt="image" src="https://github.com/user-attachments/assets/754b6aef-4214-41e9-acc5-0bf89e3d09d2">

**Comment by willpower232** [comment]
I presume it is the `dkim_identifier` https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/doc/config/yaml.yml#L145

**Reply by TR-SametOzen** [reply]
Hello,

Not working.

**Reply by TR-SametOzen** [reply]
update

ı writed postal.yml it is work. :)
